lib/puppet_x/puppetlabs/powershell/powershell_manager.rb

```diff
@@ -55,18 +55,29 @@ def initialize(cmd, debug)
         # wait for the pipe server to signal ready, and fail if no response in 10 seconds
 
         # wait up to 30 seconds in 0.2 second intervals to be able to open the pipe
-        150.times do
+        # If the pipe_timeout is ever specified as less than the sleep interval it will
```

Comment:
@Iristyle Is this comment sufficient for [this question](https://github.com/puppetlabs/puppetlabs-powershell/pull/202#discussion_r153363126)?